manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4

```

```suggestion
PackageVersion: 4.6.0.0
```
This should match control panel. The version folder will need to be renamed